### ISSUE TITLE
[CLI] IndexError: list index out of range

### ISSUE THREAD

**Issue by amirDahari1** [wandb/client#2879]
**Description**
IndexError: list index out of range when trying to import wandb. Is there some import loop maybe?

**Wandb features**
Just when I am importing and not any other code
`import wandb`

IndexError: list index out of range

At:
  /home/amir/.local/lib/python3.7/site-packages/wandb/old/core.py(26): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap>(1043): _handle_fromlist
  /home/amir/.local/lib/python3.7/site-packages/wandb/old/settings.py(6): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /home/amir/.local/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py(29): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /home/amir/.local/lib/python3.7/site-packages/wandb/apis/internal.py(1): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /home/amir/.local/lib/python3.7/site-packages/wandb/apis/__init__.py(32): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /home/amir/.local/lib/python3.7/site-packages/wandb/sdk/wandb_artifacts.py(29): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /home/amir/.local/lib/python3.7/site-packages/wandb/sdk/__init__.py(9): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap>(1043): _handle_fromlist
  /home/amir/.local/lib/python3.7/site-packages/wandb/__init__.py(34): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(728): exec_module
  <frozen importlib._bootstrap>(677): _load_unlocked
  <frozen importlib._bootstrap>(967): _find_and_load_unlocked
  <frozen importlib._bootstrap>(983): _find_and_load
  /tmp/taitherm-bEvwaQ/battery_script(3): <module>

**Environment**
- OS: Ubuntu 20.04
- Python Version: 3.7
- wandb: 0.12.6

Thanks!

**Comment by aidanjdonohue** [comment]
Hi @amirDahari1 , I wasn't able to reproduce this issue. Could you try uninstalling and reinstalling wandb to see if this helps?

**Comment by amirDahari1** [comment]
Hey @aidanjdonohue, I have tried to uninstall and reinstall, this issue is happening since the script is running through a different software (TAITherm), I am now trying to solve it with their help. I posted it here becuase I thought that it may be something that you already encountered and can be easily fixed, since other packages run smoothly and there is only problem with wandb. 
Thanks anyway for your help, I will post here again if we can't solve this problem.
